artemis-commons/src/main/java/org/apache/activemq/artemis/utils/UUIDTimer.java

```diff
@@ -293,6 +293,7 @@ private static void slowDown(final long startTime, final long actDiff) {
          if (++counter > UUIDTimer.MAX_WAIT_COUNT) {
             break;
          }
-      } while (System.currentTimeMillis() < waitUntil);
+      }
```

Comment:
actually, this should be on the same line....:/


there was a discussion an year ago more or less... I think @cshannon was part of it...